eng/common/TestResources/SubConfig-Helpers.ps1

```diff
@@ -99,6 +99,7 @@ function ShouldMarkValueAsSecret([string]$serviceName, [string]$key, [string]$va
         return $false
     }
 
+    Write-Host "Marking '$key' as secret"
```

Comment:
This should already be logged at the higher layer, though maybe I missed a couple spots.

Reply:
I was just adding this to try and identify what is marking some things secret. It didn't work so I will likely remove this.

I suspect what is marking things like tenantid as secret is coming from other rogue secrets being brought in by variable groups that we should probably eliminate.